dcc-portal-ui/app/scripts/common/js/location.js

```diff
@@ -141,6 +141,11 @@
       getPaginationParams: function(dataType){
         return _.defaults({}, this.getJsonParam(dataType), {from: 1, size: 10});
       },
```

Comment:
will rename `goToFirstPageOfDataType` to `goToFirstPage`, and `dataType` to `param`